pkg/operation/care/health_check_test.go

```diff
@@ -774,7 +774,7 @@ var _ = Describe("health check", func() {
 				checker           = care.NewHealthChecker(map[gardencorev1beta1.ConditionType]time.Duration{}, nil, nil, kubernetesVersion, gardenerVersion)
 			)
 
-			exitCondition, err := checker.CheckMonitoringControlPlane(seedNamespace, isTestingShoot, wantsAlertmanager, condition, deploymentLister, statefulSetLister)
+			exitCondition, err := checker.CheckMonitoringControlPlane(seedNamespace, isTestingShoot, true, wantsAlertmanager, condition, deploymentLister, statefulSetLister)
```

Comment:
Can you also add a test where `monitoringEnabled` is false?

Reply:
done! Please check if that is sufficient